stopes/ui/seamlisten/backend/app/fileviewer.py

```diff
@@ -131,7 +131,14 @@ def get_annotations(
 @router.post("/general/")
 async def general_query(query: DefaultQuery) -> Response:
     query_path = query.gz_path.strip()
-    print(query_path.split(" "))
+    # Convert the path to an absolute path
+    query_path = Path(os.path.expanduser(query_path)).resolve()
```

Comment:
Path does it for you, that's the benefit from using this single API.

```suggestion
    query_path = Path(query_path). expanduser().resolve()
```